keyboards/tr60w/readme.md

```diff
@@ -0,0 +1,26 @@
+# TR60W
+
+[![TR60W Imgur Link](https://imgur.com/a/vezmI99/)](https://imgur.com/a/vezmI99/)
+
+This is the hotswap pcb supplied by Triangle Lab with their tr60 case.
```

Comment:
Most of the things under Hardware Supported are better said here.

```suggestion
This is the hotswap PCB supplied by Triangle Labs with their TR60 case, featuring four RGB LEDs around the Escape key, USB-C, and SMD single-color backlighting.
```

Reply:
done.